### DISCUSSION TITLE
Podman volume export soundness

### DISCUSSION THREAD

**Discussion by IBims1NicerTobi** [containers/podman#20285] [Q&A]
Is podman volume export save to run when you have a container using the volume? Will the resulting tar archive be consistent against for example a large file being deleted during creating?

**Comment by Luap99** [comment]
No, it may not be consistent if they are writes during this operation. It is just like a simple tar on the volume directory if files are changed during this the view will not be consistent.
You would need to stop or pause the container to prevent any writes during this time.
